### PR TITLE
Dtspb 2431 ee no journey

### PR DESCRIPTION
**Pull request by itoulou**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSPB-2431

### Change description ###
No journey EE

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: app/resources/cy/translation/iht/ihtestatevalues.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    "title": "WELSH - What is the value of the estate",
+    "question": "WELSH - What are the values of the estate for Inheritance Tax?",
+    "questionHint": "If you do not have these values, use <a href=\"https://www.gov.uk/valuing-estate-of-someone-who-died\" class=\"govuk-link\" rel=\"noreferrer noopener\" target=\"_blank\">the IHT checker</a> to get them. We collect these values for HMRC so they can check if you need pay inheritance tax.",
+    "estateGrossValue": "Gross value",
+    "estateGrossValueHint": "This is the total value of all assets like property, money and gifts in the last 7 years before taking off any debts",
+    "estateNetValue": "Net value",
+    "estateNetValueHint": "Take off any debts like mortgage and credit cards from the gross value",
+    "estateNetQualifyingValue": "Net qualifying value",
+    "estateNetQualifyingValueHint": "Take off from the net value any assets passed to spouse or civil partner and money left to a charity.",
+    "poundSymbol": "£",
```
Comment (AbdullahShahHMCTS) [inline app/resources/cy/translation/iht/ihtestatevalues.json:11]: ampersand for pound symbol

File: test/component/deceased/testEnglishForeignDeathCert.js [hunk 2]
```diff
@@ -82,5 +85,44 @@ describe('english-foreign-death-cert', () => {
                     testWrapper.testRedirect(done, data, expectedNextUrlForForeignDeathCertTranslation);
                 });
         });
+
+        it(`test it DOES NOT redirects to estate valued for EE FT on: ${expectedNextUrlForIhtMethod}`, (done) => {
```
Comment (AbdullahShahHMCTS) [inline test/component/deceased/testEnglishForeignDeathCert.js:89]: duplicate test as line 98

Comment (AbdullahShahHMCTS) [reply test/component/deceased/testEnglishForeignDeathCert.js:89]: 108*

File: test/unit/iht/testDeceasedLateSpouseOrCivilPartner.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+const steps = initSteps([`${__dirname}/../../../app/steps/action/`, `${__dirname}/../../../app/steps/ui`]);
+const DeceasedHadLateSpouseOrCivilPartner = steps.DeceasedHadLateSpouseOrCivilPartner;
+
+describe('DeceasedHadLateSpouseOrCivilPartner', () => {
+    describe('getUrl()', () => {
+        it('should return the correct url', (done) => {
+            const url = DeceasedHadLateSpouseOrCivilPartner.constructor.getUrl();
+            expect(url).to.equal('/deceased-late-spouse-civil-partner');
+            done();
+        });
```
Comment (AbdullahShahHMCTS) [inline test/unit/iht/testDeceasedLateSpouseOrCivilPartner.js:14]: test next steps options

File: test/unit/iht/testIhtEstateValues.js [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+            };
+            errors = [];
+            [ctx, errors] = IhtEstateValues.handlePost(ctx, errors, {}, {language: 'en'});
+            expect(ctx).to.deep.equal({
+                estateGrossValueField: '500000',
+                estateGrossValue: 500000,
+                estateNetValueField: '400000',
+                estateNetValue: 400000,
+                estateNetQualifyingValueField: '',
+                estateNetQualifyingValue: 0.0
```
Comment (AbdullahShahHMCTS) [inline test/unit/iht/testIhtEstateValues.js:143]: Ask mac does blank = 0?

File: app/steps/ui/deceased/englishforeigndeathcert/index.js [hunk 5]
```diff
@@ -9,6 +12,15 @@ class EnglishForeignDeathCert extends ValidationStep {
         return pageUrl;
     }
 
+    next(req, ctx) {
+        const journeyMap = new JourneyMap(req.session.journey);
+        if (featureToggle.isEnabled(req.session.featureToggles, 'ft_excepted_estates') && ExceptedEstateDod.afterEeDodThreshold(ctx['dod-date']) && ctx.englishForeignDeathCert === 'optionYes') {
```
[the remark this answers is not in the document]
Comment (itoulou) [reply app/steps/ui/deceased/englishforeigndeathcert/index.js:17]: dod-date does persist in the context when you sign out, BUT there is an edge case. 

IF FT on
AND dod is > 1/1/2022
AND user logs out/exits application the page before new journey
THEN when they sign back/renter app, they continue onto the iht page, not our new page

File: app/resources/cy/translation/iht/ihtestatevalues.json (hunk 1 above)
Comment (itoulou) [reply app/resources/cy/translation/iht/ihtestatevalues.json]: Changed on all resource files

File: test/component/deceased/testEnglishForeignDeathCert.js (hunk 2 above)
Comment (itoulou) [reply test/component/deceased/testEnglishForeignDeathCert.js:89]: duplicate test removed

File: test/unit/iht/testDeceasedLateSpouseOrCivilPartner.js (hunk 3 above)
Comment (itoulou) [reply test/unit/iht/testDeceasedLateSpouseOrCivilPartner.js:14]: test added

File: test/unit/iht/testIhtEstateValues.js (hunk 4 above)
Comment (itoulou) [reply test/unit/iht/testIhtEstateValues.js:143]: Mac confirmed that is desired behaviour

File: app/steps/ui/deceased/englishforeigndeathcert/index.js (hunk 5 above)
[the remark this answers is not in the document]
Comment (nigeldunne) [reply app/steps/ui/deceased/englishforeigndeathcert/index.js:17]: Sounds like we need to add /task-list to featureToggles.js

**Review by nigeldunne — APPROVED** [review]

**Review by nigeldunne — APPROVED** [review]

**Pull request merged**